internal/cli/up.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/fatih/color"
 	"github.com/opentracing/opentracing-go"
 	"github.com/spf13/cobra"
+	engine "github.com/windmilleng/tilt/internal/engine"
```

Comment:
is this package aliasing unnecessary?

Reply:
i dunno, not sure why goimports did that